### ISSUE TITLE
Estado actual del montaje de la infraestructura de servidores

### ISSUE THREAD

**Issue by ISDanielLoaiza** [GrupoSierraNevada/planeacion-integracion#6]
Juan nos podrías por favor dar respuesta a las siguientes preguntas,

- ¿Cuál es el estado actual del montaje de la infraestructura de servidores?
- ¿Cuál es la fecha estimada para tener esta actividad finalizada?
- ¿Requieren algún apoyo por nuestra parte en este frente?

**Comment by quiasmo** [comment]
Actualmente estoy varado con la configuración d ela infraestructura de servidores. 
La razón principal estamos atrazados en pagos y solo tengo acceso a soporte cloud.

Las actividades pendientes por realizar son:
- programar una coia de nuestra BD producción a pruebas
- reinstalar re-configurar el call-center en este entorno
- reinstalar las TPVs del laboratorio para operar y probarlo

Si pueden, me pueden ayudar con esto:
- Actualmente no me han entregado un instalador o una guía para la instalación de las tpvs que me permita adecuar mi laboratorio. Si esto existe me serviría enormemente.
- Una guía para instalar la suite POS en el servidor y su IIS adecuado, aunque para la instalación del servicoi IIS si creo tener un manual.

Como carezo de estas guías y recursos, dependeo de tickets de soporte para realizar todas estas operaciones.

Me mencionan que el pago pendient está programado para el final de la semana, así que una vez tenga soporte programaré los tickets y quedaré a merced de los horarios de soporte para adecuar los 4 equpios del laboratorio.

**Comment by quiasmo** [comment]
Acabo de solicitar la copia de la BD a pruebas, pero entonces voy a perder la guía para configurar el call-center de nuevo. 😭

**Comment by ISDanielLoaiza** [comment]
Juan, en este caso lo ideal es que SIESA realice la instalación del entorno, sin embargo, consultaremos internamente si existen manuales de instalación; lo que podríamos hacer es que nos compartas el número de ticket y así nosotros poder agilizar y que este no se convierta en un limitante.

**Comment by ISDanielLoaiza** [comment]
Juan, te cuento que recién se hablo el tema con Robeiro Ramírez Director de integraciones de SIESA, como respuesta nos comento que actualmente no existe un manual de instalación, sin embargo, nos recomendó que te pusieras en contacto con Víctor Hugo Lider en consultoria de SIESA para efectos de agilizar la instalación del laboratorio.

**Comment by quiasmo** [comment]
OK, Viktor ya no es consultor de nosotros. Ahora es Lorena Rodriguez. 
Hablaré con ella.

**Comment by ISDanielLoaiza** [comment]
Efectivamente Viktor se encuentra en este momento por fuera de la compañía, sin embargo, nos comunicamos con Henry Ruiz, quien por el momento es el encargado, nos indico que para llevar a cabo la instalación del laboratorio debes seguir los siguientes pasos: 

1. Contar con la infraestructura para montar el ambiente
2. Enviar un correo electrónico a Andrés Trujillo (IT SIESA) para efectos de que él revise si se cuenta con los requisitos mínimos para realizar la instalación
3. Finalmente, generar un ticket para la instalación del ambiente

**Comment by quiasmo** [comment]
Falso, si es consultor. Me confundí.  Hubo cambio de consultor en Financiera.
